manifests/repo/redhat.pp

```diff
@@ -22,13 +20,4 @@
     gpgkey     => 'https://rpms.remirepo.net/RPM-GPG-KEY-remi',
     priority   => 1,
   }
-
-  yumrepo { 'remi-php56':
```

Comment:
Shouldn't this stay with `ensure => absent`?

Reply:
done